src/plugins/home/public/application/components/move_data/move_data.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, MouseEvent } from 'react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { METRIC_TYPE, UiCounterMetricType } from '@kbn/analytics';
+import { i18n } from '@kbn/i18n';
+import { createAppNavigationHandler } from '../app_navigation_handler';
+
+interface Props {
+  addBasePath: (path: string) => string;
+  isCloudEnabled: boolean;
+  trackUiMetric: (type: UiCounterMetricType, eventNames: string | string[], count?: number) => void;
+}
+
+export const MoveData: FC<Props> = ({ addBasePath, isCloudEnabled, trackUiMetric }) => {
+  const migrateDataUrl = !isCloudEnabled
```

Comment:
nit: I think using a negated boolean is little less readable than using it directly, something like 
```suggestion
  const migrateDataUrl = isCloudEnabled ? 'https://ela.st/cloud-migration' : '/app/management/data/migrate_data'
```